service/controller/resource/instancestatus/delete.go

```diff
@@ -0,0 +1,16 @@
+package instancestatus
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+)
+
+func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
+	err := r.ensure(ctx, obj)
```

Comment:
Is it really necessary to run this also on cluster deletion? I would like to make sure this does not create too much of a struggle. In the past we sometimes got into situations where deletion was completely broken because some resource implementation was buggy.

Reply:
Will remove :+1: